gems/smithy/lib/smithy/anvil/client/templates/shapes.erb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+module <%= namespace %>
+  module Shapes
+    include Smithy::Client::Shapes
+
+<% shapes.each do |shape| -%>
+    <%= shape.name %> = <%= shape.type %>.new(id: "<%= shape.id %>", traits: <%= shape.traits %>)
+<% end -%>
+
+<% shapes_with_members.each do |shape| -%>
+<% shape.members.each do |member| -%>
```

Comment:
One improvement you can make here is push this logic to the view class. These are very easy strings you can prepare.

Reply:
Yeah I think I figured out a way to push the logic there so the .erb looks cleaner.